creator/studies/mutations.py

```diff
@@ -175,6 +180,12 @@ def mutate(self, info, input, workflows=None):
         if not user.has_perm("studies.add_study"):
```

Comment:
Actually this may require more thought. We'll figure this out with a new RFC I'm planning to write for the permission stuff.
For now, I think maybe I'll restrict it to the user must have `add_study` AND belong to the organization.

There may be an administrative case where we want an admin to create a study without being in an org, but I believe this can be addressed through some of the permission refactoring.